docs/basics/101-125-siblings.rst

```diff
@@ -0,0 +1,190 @@
+   :workdir: dl-101/mock_user/DataLad-101
+
+   $ datalad save -m "add shell script from ascii cast on nesting" code/nested_repos.sh
+
+Suddenly, your room mate has a file change that you do not have.
+His dataset evolved.
+
+So how do we link back from the copy of the dataset to its
+origin, such that your room mates changes can be included in
+your dataset? How to we let the original dataset "know" about
+this copy your room mate has?
+Do we need to install the installed dataset of our room mate
+as a copy again?
+
+No, luckily, it's simpler and less convoluted. What we have to
+do is to create a datalad ``sibling``: A reference to our room mates
+dataset in our own, original dataset.
+
+.. gitusernote::
+
+   Git repositories can configure clones of a dataset as *remotes* in
+   order to fetch, pull, or push from and to them. A ``datalad sibling``
+   is the equivalent of a git clone that is configured as a remote.
+
+Let's see how this is done.
+
+First of all, navigate back into the original dataset.
+In the original dataset, "add" a "sibling" by using
+the ``datalad siblings`` command. The command takes the base command,
+``datalad siblings``, an action, in this case ``add``, a path to the
+root of the dataset ``-d .``, a name for the sibling, ``-s/--name roommate``,
+and a URL or path to the sibling, ``--url ../mock_user/DataLad-101``.
+This registers your roommates ``DataLad-101`` as a "sibling" (we will call it
+"roommate") to your own ``DataLad-101`` dataset.
+
+
+.. runrecord:: _examples/DL-101-125-104
+   :language: console
+   :workdir: dl-101/mock_user/DataLad-101
+
+   cd ../../DataLad-101
+   # add a sibling
+   $ datalad siblings add -d . --name roommate --url ../mock_user/DataLad-101
+
+There are a few confusing parts about this command: For one, don't be surprised
+about the ``--url`` argument -- it's called "URL" but it can be a path as well.
+Also, don't forget to give a name to your datasets sibling. Without the ``-s``/
+``--name`` argument the command will fail. The reason behind this is that the default
+name of a sibling if no name is given will be the hostname of the specified URL,
+but as you provide a path and not a URL, there is no hostname to take as a default.
+
+As you can see in the command output, the addition of a sibling succeeded:
+``roommate(+)[../mock_user/DataLad-101]`` means that your room mates dataset
+is now known to your own dataset as "roommate"
+
+
+.. runrecord:: _examples/DL-101-125-105
+   :language: console
+   :workdir: dl-101/mock_user/DataLad-101
+
+   $ datalad siblings
+
+This command will list all known siblings of the dataset. You can see it
+in the resulting list with the name "roommate" you have given to it.
+
+The fact that the ``DataLad-101`` dataset now has a sibling means that we
+can also ``datalad update`` this repository:
+
+.. runrecord:: _examples/DL-101-125-106
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad update --merge -s roommate
```

Comment:
Consider not going for the `--merge` right away, but plain `update` first, then `datalad diff` against the remote, see the coming change, then merge.